near-accounts/examples/function_call.rs

```diff
@@ -1,22 +1,23 @@
 use near_accounts::Account;
 use near_crypto::InMemorySigner;
+use near_crypto::SecretKey;
 use near_primitives::types::Gas;
 use near_providers::JsonRpcProvider;
 use std::sync::Arc;
 mod utils;
 use near_primitives::types::AccountId;
 use serde_json::json;
+use tokio::time;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     env_logger::init();
 
-    let signer_account_id: AccountId = utils::input("Enter the signer Account ID: ")?.parse()?;
-    let signer_secret_key = utils::input("Enter the signer's private key: ")?.parse()?;
+    let signer_account_id: AccountId = "near-api-rs.testnet".parse::<AccountId>()?;
+    let signer_secret_key = "ed25519:29nYmQCZMsQeYtztXZzm57ayQt2uBHXdn2SAjK4ccMGSQaNUFNJ7Aoteno81eKTex9cGBbk1FuDuqJRsdzx34xDY".parse::<SecretKey>()?;
```

Comment:
Is this intentional? if yes then I guess the test user details we can add in some resource file.

Reply:
Applicable for all the other examples as well.